File: 20.py
def factorial(n):
	if(n == 1):
		return 1;
	else:
		return n * factorial(n-1)

def sum_digits(n):
	sum = 0
	for digit in n:
		# print(digit)
		sum += int(digit)
	return sum

print(sum_digits(str(factorial(100))))

File: 20.js

function factorial(n) {
if(n === 1){
	return 1;
} else {
	return (n * factorial(n - 1)).toLocaleString('fullwide', {useGrouping:false}) ;
}
}

function sumOfDigits(n){
	let sum = 0;
	for(let i = 0; i < n.length; i++){
		// console.log(i)
		sum += parseInt(n[i]);
	}
	return sum;
}
console.log(factorial(100))

console.log(sumOfDigits(factorial(100)));